src/components/Calculator.js

```diff
@@ -1,34 +1,55 @@
 import React from 'react';
 import './Calculator.css';
+import CalcButton from './buttons';
+import calculate from '../logic/calculate';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
I tried removing the line but it will show a linter error and there is no way in changing it because of how react is and the requirements for the project.